### ISSUE TITLE
I can backup to USB but not to Nextcloud

### ISSUE THREAD

**Issue by ElucGeek** [seedvault-app/seedvault#274]
Hello,
I've just updated my Mi Mix 2 (Chiron) to LineageOS 18.1 (Android 11) and after spending one day to reinstall and restore every app manually, I was very happy to discover this SeedVault backup feature. 
Thus I had some issues to backup to Nextcloud. It detect the app and write some hidden folders and files in it during the initialization but as soon as as start a backup it fail to run. If I select back to USB it mostly work (some apps fail but most worked).
Any idea how to debug in order to allow Nextcloud backup?

**Comment by adymorz** [comment]
I encounter also problems to backup data to nextcloud, and also backing up to an USB stick works (except for the app which do not allow the data to be backed up).

I suspected that the error was coming from the fail2ban module, which is set up on the reverse proxy before the nextcloud instance. To check this, I started the backup on two mobile phones and waited for the error. To see if and when the error occurs, I activated "Caffeine" (https://android.gadgethacks.com/how-to/get-cyanogenmods-caffeine-feature-keep-your-screen-awake-longer-press-button-0176126/) to prevent the screen from being locked and to make sure that I immediately recognise when the process is interrupted.

On one mobile phone, the backup went through without issues. On the other phone it got stuck again, showing a notification from SeedVault App "Backup failed" (german: "Sicherung fehgeschlagen") and also showing a nofication from Nextcloud app "Downloading" (german: "Lader herunter..."), "100% downloaindg MTUw" (german: "100% Herunterladen MTUw"). fail2ban had no active jail after the backup got stuck, which means, that fail2ban does not seem to be the cause for the issues here.
